src/read_agent_logging.in

```diff
@@ -20,22 +20,32 @@ URL="${1:-http://localhost:24231/metrics}"
 INTERVAL=${COLLECTD_INTERVAL:-10}
 
 # The part of the input that sed looks at looks like:
-#   stackdriver_successful_requests_count{grpc="false"} 255
-#   stackdriver_failed_requests_count{grpc="false",code="400"} 14
+# 1. If the Logging Agent uses REST.
+#   stackdriver_successful_requests_count{grpc="false",code="200"} 255
 #   stackdriver_failed_requests_count{grpc="false",code="404"} 18
-#   stackdriver_ingested_entries_count 7050
-#   stackdriver_dropped_entries_count 25
-#   stackdriver_retried_entries_count{code="500"} 2
-#   stackdriver_retried_entries_count{code="501"} 3
-#
+#   stackdriver_ingested_entries_count{grpc="false",code="200"} 7050
+#   stackdriver_dropped_entries_count{grpc="false",code="401"} 25
+#   stackdriver_retried_entries_count{grpc="false",code="500"} 2
+# 2. If the Logging Agent uses gRPC.
+#   stackdriver_successful_requests_count{grpc="true",code="0"} 255
+#   stackdriver_failed_requests_count{grpc="true",code="3"} 18
+#   stackdriver_ingested_entries_count{grpc="true",code="0"} 7050
+#   stackdriver_dropped_entries_count{grpc="true",code="3"} 25
+#   stackdriver_retried_entries_count{grpc="true",code="13"} 2
+
 # The expected output follows:
+# 1. If the Logging Agent uses REST.
 #   PUTVAL /agent-200/derive-request_count N:255
-#   PUTVAL /agent-400/derive-request_count N:14
 #   PUTVAL /agent-404/derive-request_count N:18
 #   PUTVAL /agent-200/derive-log_entry_count N:7050
-#   PUTVAL /agent-418/derive-log_entry_count N:25
+#   PUTVAL /agent-401/derive-log_entry_count N:25
 #   PUTVAL /agent-500/derive-log_entry_retry_count N:2
-#   PUTVAL /agent-501/derive-log_entry_retry_count N:3
```

Comment:
You could keep this — it translates to gRPC code 12...

Reply:
I removed this one because having one example for `log_entry_retry_count` should be enough. Having two feels redundant. WDYT?

Reply:
I think it's good to have that to illustrate that there's more than one error code that's retried. We should have multiple examples of failed requests as well.

Reply:
Added more examples.